### DISCUSSION TITLE
Welcome to predictr Discussions!

### DISCUSSION THREAD

**Discussion by tvtoglu** [tvtoglu/predictr#2] [Announcements]
## 👋 Welcome!
  Let's use Discussions as a place to connect with other members of our community. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by maheshyalagach** [comment]
Hello, 

Had a question on the following:

 How do you compute the pval while using the MRR method to analyse the Weibull Distribution ?
Do you use an AD method. Can you please explain the significance of this pValue. 

BR

Mahesh

**Reply by tvtoglu** [reply]
Hello Mahesh,

the p-value is computed by using the Wald Test (t-distribution). If the null hypothesis can't be rejected, the slope (here: Weibull shape parameter) is zero (non-zero or more precisely >0 for the alternative hypothesis).

If the p-value is low, the null hypothesis can be rejected. Thus, the alternative hypothesis is true, i.e. that the test statistic is likely not zero. This is the standard for linear regression in the scipy package. I hope this information helps.

predictr does not use the AD test. But I might add some alternative goodness-of-fit test in future. 

Best regards
